Rain Alert Mail Sender/main.py

```diff
@@ -0,0 +1,48 @@
+import requests
+from smtplib import SMTP
+
+#turn off certain security criteria in sender mail address
+MY_MAIL= ENTER_YOUR_MAIL
+MY_PASSWORD= ENTER_YOUR_PASSWORD
+RECIEVER_MAIL = RECIEVER_MAIL
+#get latitude longitude from  "https://www.latlong.net/"
+LAT = 22.572645
+LONG = 88.363892
```

Comment:
What are these coordinates for? If you're using them for location then get the location from the user and then use `https://nominatim.openstreetmap.org/search/` to get the exact coordinates of the location